### DISCUSSION TITLE
qBT Login Failing after upgrade

### DISCUSSION THREAD

**Discussion by SawkeeReemo** [cross-seed/cross-seed#324] [Q&A] [answered]
Hey, all. Been a minute, hope you're well!

I'm still running cross-seed v3.4.2 which has been working perfectly for me... until I upgraded qBT from 4.4.2 to 4.4.5. Now all of a sudden it's rejecting my login info. I jumped back to qBT 4.4.2 and cross-seed works again. The error I'm getting is:
`qBittorrent login failed with code 405`

Any idea what's up there? Would love to be able to update qBT but keep cross-seed running.
Thanks!

**UPDATE:** I'm seeing this was addressed in a later update [v4.3.6](https://github.com/cross-seed/cross-seed/releases/tag/v4.3.6).

**Comment by evan326** [comment]
I'm having this issue also. How were you able to update cross-seed to a higher version? 
Pulling the latest or master tag both pull down 4.3.0 still

**Reply by bakerboy448** [reply]
Sounds like you're not actually updating the container and or are using a legacy unsupported docker container

https://www.cross-seed.org/docs/basics/getting-started#with-docker-recommended

**Reply by evan326** [reply, marked as answer]
it seems that mmgoodnow/cross-seed isn't being updated anymore? going with crossseed/cross-seed works. I guess I missed the switch?
thanks

**Reply by Jerrk** [reply]
> it seems that mmgoodnow/cross-seed isn't being updated anymore? going with crossseed/cross-seed works. I guess I missed the switch? thanks

i definitely also missed this change, seems to have been mentioned in the changelog for 4.3.1 https://github.com/cross-seed/cross-seed/releases/tag/v4.3.1
